src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,6 +1,66 @@
 package com.manardenza.utils;
 
-public class DatabaseSeeder {
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
 
-    // Class to fill DAO implementations
+import java.util.ArrayList;
+import java.util.List;
+
+public final class DatabaseSeeder {
+
+    private static final String KIEV = "Kiev";
+    private static final String ODESSA = "Odessa";
+    private static final String LVOV = "Lvov";
+    private static final String DONETSK = "Donetsk";
+    private static final String ZAPOROZHYE = "Zaporozhye";
+    private static final String MARRIOTT = "Marriott";
+    private static final String HOLIDAY_INN = "Holiday Inn";
+    private static final String REIKARTZ = "Reikartz";
+
+    private static List<Room> roomsSeederList = new ArrayList<>();
+
+    public static void seedDatabase() {
+        UserDaoImpl.getInstance().save(new User(464655, "Vasya", "Petrov"));
```

Comment:
You may call UserDaoImpl.getInstance() only once and store result in a local variable